core/controllers/creator_dashboard.py

```diff
@@ -276,7 +272,7 @@ def _round_average_ratings(rating):
             [('author_id', self.user_id),
              (
                  'suggestion_type',
-                 suggestion_models.SUGGESTION_TYPE_EDIT_STATE_CONTENT)])
+                 feconf.SUGGESTION_TYPE_EDIT_STATE_CONTENT)])
```

Comment:
Just a question about the design. Would it make sense to have a constants file specific to the feature similar to how we have in the frontend? Might help scope down many of the constants in feconf.

Reply:
I guess after refactoring of the codebase we will use domain object files to also store the constants related to that domain.